aws-datastore/src/test/java/com/amplifyframework/datastore/appsync/TestModelWithMetadataInstances.java

```diff
@@ -53,12 +61,18 @@
                 .rating(3)
                 .id("83ceb757-c8c8-4b6a-bee0-a43afb53a73a")
                 .build(),
-            new ModelMetadata("83ceb757-c8c8-4b6a-bee0-a43afb53a73a", null, 5, 123123L)
+            new ModelMetadata("83ceb757-c8c8-4b6a-bee0-a43afb53a73a",
+                    null,
+                    5,
+                    new Temporal.Timestamp(123123L, TimeUnit.SECONDS))
         );
     public static final ModelWithMetadata<Post> DELETED_DRUM_POST =
         new ModelWithMetadata<>(
             DRUM_POST.getModel(),
-            new ModelMetadata("83ceb757-c8c8-4b6a-bee0-a43afb53a73a", Boolean.TRUE, 5, 123123L)
+            new ModelMetadata("83ceb757-c8c8-4b6a-bee0-a43afb53a73a",
+                    Boolean.TRUE,
+                    5,
+                    new Temporal.Timestamp(123123L, TimeUnit.SECONDS))
```

Comment:
We could potentially create `Temporal.Timestamp.seconds(123_123L)` and `Temporal.Timestamp.milliseconds(123_123_000L)` utility factories?